src/main/java/uk/gov/hmcts/divorce/divorcecase/tab/CaseTypeTab.java

```diff
@@ -231,7 +230,7 @@ private void buildConfidentialApplicantTab(ConfigBuilder<CaseData, State, UserRo
     private void buildConfidentialRespondentTab(ConfigBuilder<CaseData, State, UserRole> configBuilder) {
         configBuilder.tab("ConfidentialRespondent", "Confidential Respondent")
             .forRoles(CASE_WORKER, LEGAL_ADVISOR, JUDGE, APPLICANT_2_SOLICITOR, SUPER_USER)
-            .showCondition("applicant2ContactDetailsType=\"private\" AND applicationType=\"soleApplication\"")
+            .showCondition("applicationType=\"soleApplication\"")
```

Comment:
EXUI users who have permission to access contact details would need to start viewing it through the confidential tabs, which will be available by default, regardless of whether contact details are set to private. If we agree this approach looks reasonable, I'll ask for approval from the service team, as they may need to inform CTSC that they will need to start viewing this tab for address information